equihash/equihash.go

```diff
@@ -0,0 +1,722 @@
+package consensus
+
+import (
+	"bytes"
+	"crypto/sha256"
+	"encoding/binary"
+	"errors"
+	"hash"
+	"math"
+	"math/big"
+	"reflect"
+	"sort"
+
+	blake2b "github.com/minio/blake2b-simd"
```

Comment:
Why not standard golang "crypto" package?

Reply:
The stdlib version of blake2b doesn't provide all blake2b parameters needed to customize the hash.

Reply:
Good spot! I just added it [now] (https://github.com/EXCCoin/exccd/pull/11/commits/8da7052ecd186e97d4c8e66d0c0be9525ae392ac)